File: C++/Array/1672 _Richest_Customer_Wealth.cpp
#include <iostream>
using namespace std;
#include <vector>

int SumElements(vector<vector<int>> array2)
{
    int MaxWealth= 0;

    for (int i = 0; i < array2.size(); i++) {

        int Wealth = 0;

        for (int j = 0; j < array2[i].size(); j++) {
            Wealth +=  array2[i][j];
        }

        if (Wealth >= MaxWealth) {
            MaxWealth = Wealth;
        }
    }

    return MaxWealth;
}

int main()
{
    vector<vector<int>> FindElement = {
        {1, 2, 3},
        {1, 2, 4},
        {10,20}
    };

    int Result = SumElements(FindElement);

    cout << "result: "<< Result;

    return 0;
}

File: Python/Array/1672 _Richest Customer Wealth.py
class Solution:
    def SumElements(self, ElementArray):
        Max_Wealth = 0

        for i in range(len(ElementArray)):
            Wealth = 0

            for j in range(len(ElementArray[i])):
                Wealth += ElementArray[i][j]

            if Wealth >= Max_Wealth:
                Max_Wealth = Wealth


        return Max_Wealth

def main():
    solution = Solution()
    ElementArray = [[1,2,4],[1,2,10],[10,20]]

    result = solution.SumElements(ElementArray)

    print("result: \n", result)

if __name__ == "__main__":
    main()